cmd/buildah/images.go

```diff
@@ -13,6 +18,23 @@ type jsonImage struct {
 	Names []string `json:"names"`
 }
 
+type imageOutputParams struct {
+	ID        string
+	Name      string
+	Digest    string
+	CreatedAt string
+	Size      string
+}
+
+type filterParams struct {
+	dangling         string
+	label            string
+	beforeImage      string // Images are sorted by date, so we can just output until we see the image
+	sinceImage       string // Images are sorted by date, so we can just output until we don't see the image
```

Comment:
The storage library doesn't take steps to ensure anything about the ordering of images that it returns, and it's probably a bug that the creation date that the image library metadata records for a pulled image is the time at which the image was pulled -- if the image's config blob includes a datestamp indicating when it was built on whichever system it was built, I think we'd want to use that value instead.